src/main/java/editQuiz/EditQuizOutputData.java

```diff
@@ -9,16 +9,27 @@
 
```

Comment:
for consistency put the description above the class name as a javadoc

Reply:
Another day.